aspnetcore/includes/razor.md

```diff
@@ -1,9 +1,10 @@
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a feature of the ASP.NET Core MVC framework that makes building and testing web UI easier and more productive. You can use Razor pages alongside controllers and views in the same project.
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial.
 
-We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC/Controller/Views version. The Razor Pages tutorial:
+Razor Pages:
 
-* Is the preferred approach for new application development.
-* Is easier to follow.
-* Covers more features.
+* Are part of the ASP.NET Core MVC framework. You can mix Razor Pages with controllers and views.
+* Programming is very similar to MVC programming. Most of the ASP.NET Core APIs and features can be used with Razor Pages or controllers with views.
+* Many developers find Razor Pages easier to use and maintain.
```

Comment:
This sounds like marketing, but without it you don't have anything to say about why it might be better; would be good if you could include some objective reason why someone might choose RP, and then make that first in the list

Reply:
@tdykstra 	need #7801 to merge.

Reply:
>need #7801 to merge.

Can you address any comments that you agree with on 7801, merge it, and then do another iteration on it, taking into account changes made here?

BTW, ignore my "would be good if you could include some objective reason" if you go with the recommendations to drop the bullet points and add at the end "...of this tutorial for a easier starting point"

Reply:
Dan owns #7801